templates/security/cves/cve.html

```diff
@@ -527,32 +535,72 @@ <h3>Other references</h3>
     </div>
   </section>
 
-  <!-- Linting is turned off as additional indentaion on line 533 will break the rest of the script -->
+  <!-- Linting is turned off as additional indentaion on view variables (maintainedCount, isOnlyUpstream) will break the rest of the script -->
   {# djlint: off #}
   <script>
     function handleHiddenReleasesSwitch() {
       const maintainedCount = {{ maintained_count | tojson | safe }};
+      const isOnlyUpstream = {{ only_upstream | tojson | safe }};
       const maintainedReleasesSwitch = document.querySelector(".js-toggle-maintained");
-      let hiddenCells = document.querySelectorAll(".js-unmaintained-cell");
+      const hiddenCells = document.querySelectorAll(".js-unmaintained-cell");
 
       // When the CVE does not include statuses for maintained releases, the switch is disabled and checked
-      if (maintainedCount === 0) {
+      if (maintainedCount === 0 || isOnlyUpstream) {
```

Comment:
Do you need to update the above comment to reflect this addition?

Reply:
Yes, thanks for catching!